code/ai/ai_profiles.cpp

```diff
@@ -568,7 +565,6 @@ void parse_ai_profiles_tbl(const char *filename)
 
 				//Intention is to expand this feature to include a preference for close or long range weapons
 				//hence using something besides a simple flag.
-				profile->ai_range_aware_secondary_select_mode = AI_RANGE_AWARE_SEC_SEL_MODE_RETAIL;
```

Comment:
In my defense, I didn't realize this was the one table that looped! 😉 Funny, since I've recently been using this table as a role model and never noticed this goof since...